src/VCTwoFluidStaggeredStokesBoxRelaxationFACOperator.cpp

```diff
@@ -1035,6 +1041,7 @@ VCTwoFluidStaggeredStokesBoxRelaxationFACOperator::performProlongation(const std
 
     refine_alg.resetSchedule(d_prolong_scheds[dst_ln]);
     d_prolong_scheds[dst_ln]->fillData(d_new_time);
+    d_prolong_alg->resetSchedule(d_prolong_scheds[dst_ln]);
```

Comment:
This and the following changes fix a bug when the patch indices in `d_prelong_scheds[dst_ln]` are removed from the `VariableDatabase`, but the schedule is not changed.